spec/unit/database/database.spec.js

```diff
@@ -1,14 +1,13 @@
-var db_config = require("../../../src/config/config"),
-    path = require("path");
+var dbConfig = require("../../../src/config/config"),
+  path = require("path");
 describe("Database", function() {
-    describe("Application", function() {
-        beforeEach(function() {
-            var node_env = process.env.NODE_ENV,
-                databaseURI = process.env.KIPIT_DATABASE_URI,
-                databaseName = path.basename(databaseURI);
-        });
-        it("should be connected to the correct database.", function() {
-            expect(db_config[node_env].database).toEqual(databaseName);
-        });
+  describe("Application", function() {
+    beforeEach(function() {
+        databaseURI = process.env.KIPIT_DATABASE_URI,
+        databaseName = path.basename(databaseURI);
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>'databaseName' is not defined.<br>'databaseURI' is not defined.